lib/ews/connection.rb

```diff
@@ -225,6 +225,8 @@ def check_response(resp, include_http_headers: false)
       raise Errors::UnhandledResponseError.new("Unhandled HTTP Redirect", resp)
     when 401
       raise Errors::UnauthorizedResponseError.new("Unauthorized request", resp)
+    when 429
+      raise Errors::TooManyRequestsError.new("TooManyRequests", resp)
```

Comment:
Could we change `"TooManyRequests"` to `"Too Many Requests"`? Im thinking the message should ideally be human readable